$NetBSD: patch-ak,v 1.12 2019/06/18 12:38:36 nia Exp $

Disable logo demuxer detection, it doesn't work and stops
us from building. In practice, not having it isn't a problem.

--- configure.orig	2017-12-27 19:53:11.000000000 +0000
+++ configure
@@ -19817,39 +19817,8 @@ else
   withval=auto
 fi
 
-LOGO_FORMAT=''
-logo_sub=''
-while test "$LOGO_FORMAT" = ''; do
-  echo "$as_me:$LINENO: trying type=$withval" >&5
-  if test "$withval" = auto; then
-    # preferred is last
-    if test "$HAVE_IMAGE" = yes; then withval=image; fi
-    if test "$HAVE_MPEG" = yes; then withval=mpeg; fi
-    # handle missing-demuxer case
-    if test "$withval" = auto; then
-      as_fn_error $? "no available demuxer for the logo image - rebuild/reinstall xine-lib?" "$LINENO" 5
-    fi
-  fi
-  case "$withval" in
-    image)
-	if test "$HAVE_IMAGE" = no; then
-	  logo_sub="$withval"
-	  withval=auto
-	else
-	  LOGO_FORMAT=jpg
-	fi
-	;;
-    mpeg)
-	if test "$HAVE_MPEG" = no; then
-	  logo_sub="$withval"
-	  withval=auto
-	else
-	  LOGO_FORMAT=mpv
-	fi
-	;;
-  esac
-done
-
+LOGO_FORMAT='mpv'
+logo_sub='mpeg'
 
 cat >>confdefs.h <<_ACEOF
 #define LOGO_FORMAT "$LOGO_FORMAT"
